UIs/pages/login.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Log In
+        </title>
+
+    </head>
+
+
+    <body>
+
+        <form id="theForm" action="" method="">
+
+            <!-- <label for="emailorusername">Email or Username</label> -->
+            <input type="text" placeholder="email or username" id="Username" name="">
+
+            <!-- <label for="password">Password</label> -->
+            <input type="text" placeholder="password" id="password" name="">
+
+            <input class="button" type="submit" value="Log In">
+
+            <h5>Not yet registered? Create your account</h5>
```

Comment:
On this one, You can make the `Create your account` a link because eventually it should point to the `sign up` page

Reply:
Noted with thanks